packages/framer-motion/src/projection/node/create-projection-node.ts

```diff
@@ -1245,6 +1264,7 @@ export function createProjectionNode<I>({
                  * a render to ensure the elements reflect the removed transform.
                  */
                 if (this.projectionTransform) {
+                    this.projectionDelta = createDelta()
```

Comment:
Why do we need this?

Reply:
And do we need to reset `this.projectionDeltaWithTransform` as well?

Reply:
Basically by setting `projectionTransform` to `"none"` we want to reset the underlying data structure to (`createDelta` returns a none transform)